dev/cf_operator/BUILD.bazel

```diff
@@ -1,25 +1,28 @@
+load(
+    "//rules/helm:def.bzl",
+    helm_upgrade = "upgrade",
+    helm_delete = "delete",
+)
+load("//:def.bzl", "project")
+
 package(default_visibility = ["//visibility:public"])
 
-load("//rules/helm:def.bzl", helm_template = "template")
-load("//rules/kubectl:def.bzl", kubectl_apply_binary = "apply", kubectl_delete_binary = "delete")
-load("//:def.bzl", "project")
+INSTALL_NAME = "cf-operator"
 
-helm_template(
-    name = "rendered_template",
-    values = glob(["**/*values.yaml"]),
-    install_name = "cf-operator",
-    namespace = project.namespace,
+helm_upgrade(
+    name = "install_or_upgrade",
```

Comment:
Doesn't `apply` still work? The name here is kind of long…

Reply:
`apply` was a reference to `kubectl apply`. Do you think it's fine to keep it here?

Reply:
I think that's still fine, since it's logically doing the same thing. It applies the correct configuration.